src/scaladoc/scala/tools/nsc/doc/model/ModelFactory.scala

```diff
@@ -886,7 +886,7 @@ class ModelFactory(val global: Global, val settings: doc.Settings) {
           // units.filter should return only one element
           (currentRun.units filter (_.source.file == aSym.sourceFile)).toList match {
             case List(unit) =>
-              // SI-4922 `sym == aSym` is insufficent if `aSym` is a clone of symbol
+              // SI-4922 `sym == aSym` is insufficient if `aSym` is a clone of symbol
               //         of the parameter in the tree, as can happen with type parametric methods.
```

Comment:
Changed to `as can happen with type parameterized methods`. @som-snytt - is this correct?